weDevsOfficial/dokan
# update: separate shipping tax from total tax

### All Submissions:






### Changes proposed in this Pull Request:



![Screenshot 2023-04-04 at 11 17 10 AM](https://user-images.githubusercontent.com/5476784/229698432-0b52a5f6-933c-4f0e-9b71-4e268299aabd.png)

* I've added new admin settings for *Shipping Tax Fee Recipient*.
* Changed "Tax Fee Recipient" to "Product Tax Fee Recipient".
* Added necessary help text.
* Those Closes #1384.

### How to test the changes in this Pull Request:

1. Check issue #1384 for more pieces of information.

Related Dokan Pro PR: https://github.com/weDevsOfficial/dokan-pro/pull/2280

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

**update**: Shipping fee recipient admin settings added.


### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.
